### DISCUSSION TITLE
CYBERCHEF

### DISCUSSION THREAD

**Discussion by ekky19** [Security-Onion-Solutions/securityonion#7722] [Unsupported Versions] [answered]
I have been following this Analysis https://blog.securityonion.net/2022/02/quick-malware-analysis-emotet-epoch-5.html

When I send the network.data.decoded part to cyberchef I don't get a result.

This is mine:
![image](https://user-images.githubusercontent.com/6872571/161878850-ea7e77c6-2aa1-4391-bdc8-6f095bd06d59.png)

And this is from the example:
![image](https://user-images.githubusercontent.com/6872571/161878899-8b69ff77-b3a3-464f-8649-00a7097b0b11.png)

Howcome the input looks different in the example... i couldnt figure out.

**Comment by dougburks** [comment, marked as answer]
Instead of sending the `network.data.decoded` field, you'll want to pivot to PCAP and then send the entire stream to CyberChef:
https://docs.securityonion.net/en/2.3/alerts.html#actions
https://docs.securityonion.net/en/2.3/pcap.html#pcap
https://docs.securityonion.net/en/2.3/cyberchef.html#accessing
